contrib/action_nitro/src/action_nitro.nit

```diff
@@ -561,7 +561,7 @@ redef class Player
 		end
 
 		# Display respawn instructions
-		app.ui_sprites.add new Sprite(app.texts_sheet.respawn, app.ui_camera.center)
+		app.ui_sprites.add new Sprite(app.texts_sheet.respawn, app.ui_camera.center.offset(0.0, 0.0, 0.0))
```

Comment:
It make sense then. Maybe document this usage in the method offset that an usage could be to transform (concertize?) a IPoint3d as a real Point?